components/CandidateList/CandidateList.tsx

```diff
@@ -0,0 +1,20 @@
+﻿import React from "react";
+import scss from './CandidateList.module.scss';
+import {getCandidates} from "../../api/candidatesApiClient";
+import {CandidateLink} from "../CandidateLink/CandidateLink";
+
+
+export function CandidateList(): JSX.Element {
+
+    //TODO: use getCandidates function to get actual data
+    
+    const candidates: string[] = ["Michael Walker", "Stephen Jackson"];
+    
+    return (
+        <ul className={scss.list}>
+            {
+                candidates.map(name => <CandidateLink key={candidates.indexOf(name)} name={name}/>)
```

Comment:
js gives us a neater syntax for this! :) 
(or once you have the Id you can just use that)

```ts
candidates.map((name, index) => <CandidateLink key={index} name={name}/>)
```

Reply:
used index for this